rustls/src/server/handy.rs

```diff
@@ -1,3 +1,12 @@
+use alloc::string::{String, ToString};
+use alloc::sync::Arc;
+use alloc::vec::Vec;
+use std::collections;
```

Comment:
Pre-existing issue, but it seems more consistent with our import style to import `HashMap` directly rather than importing `collections` here.

Reply:
Fixed in a separate commit at the end.